test/signing/index.ts

```diff
@@ -107,25 +107,25 @@ describe('Test General Signing', () => {
     global.test.continue = true;
   });
 
-  it('Should load determinism tests', async () => {
-    require('./determinism');
-    global.test.continue = true;
-  });
+  // it('Should load determinism tests', async () => {
```

Comment:
Also pls get rid of this cruft

Reply:
[92a00ff](https://github.com/GridPlus/gridplus-sdk/commit/92a00ff6f13e73938e84bac246d147e0129be9de)